packages/amplify-cli/src/amplify-exception-handler.ts

```diff
@@ -47,21 +47,34 @@ export const handleException = async (exception: unknown): Promise<void> => {
     printAmplifyException(amplifyException);
   }
 
+  // Swallow and continue if any operations fail
```

Comment:
Should we just have one big `try-catch` with good explanation why are we doing this?

Reply:
I originally had a single try/catch, but then I decided it would be better to have more granular error messages as well as let each operation attempt to run independent of whether or not the other calls succeeded. I can move to using private function though - it would be cleaner.